cli/README

```diff
@@ -1,5 +1,10 @@
-May need to install requests (pip install requests). 
+First install dependencies: 'pip install -r requirements.txt'
 
-Drop it in your $PATH, and do 'cat file | zifbin'. To set expiration, do a '-e <time-in-seconds>'. 
+To use this:
 
-Drop .zifbin.rc in $HOME, edit as needed
+Drop 'zifbin' in your $PATH, and do 'zifbin < ./path/to/myfile'. Run
+'zifbin -h' for other options.
+
+Drop .zifbin.rc in $HOME, edit as needed. If you hate polluting your
+home directory, XDG base directory is also supported; try
+'~/.config/zifnab/config.rc'.
```

Comment:
Typo (err, brain-o): It says "zifnab" instead of "zifbin". (The code uses "zifbin", afaict.)